edits from AD review (#173)
danielfett/draft-dpop: main.md

JOSE header of a DPoP JWT MUST contain at least the following parameters:

 * `typ`: with value `dpop+jwt`, which explicitly types the DPoP proof JWT as recommended in [@RFC8725, section 3.11].

>  * `alg`: a JWS algorithm identifier, from the registered algorithms [@IANA.JOSE.ALGS], of an asymmetric digital signature.

Reviewer: Maybe like this?
```suggestion
 * `alg`: an identifier for a JWS asymmetric digital signature algorithm from [@IANA.JOSE.ALGS].
```

Author: wfm

Note: To accommodate for clock offsets, the server MAY accept DPoP
proofs that carry an `iat` time in the reasonably near future (on the order of seconds or minutes).
Because clock skews between servers
and clients may be large, servers MAY limit DPoP proof lifetimes by using
server-provided nonce values containing the time at the server rather than
comparing the client-supplied `iat` time to the time at the server.  Nonces
created in this way yield the same result even in the face of arbitrarily
large clock skews.

Server-provided nonces are an effective means for further reducing the chances for successful DPoP proof replay.
Unlike cryptographic nonces, it is acceptable for clients to use the same 
`nonce` multiple times, and for the server to accept the same nonce multiple
times. As long as the `jti` value is tracked for the lifetime of the `nonce`, there

> is no additional risk of token replay.

Reviewer: I'm not sure if 'tracked' is enough here?

Reviewer: Sounds good!

## DPoP Proof Pre-Generation {#Pre-Generation}

An attacker in control of the client can pre-generate DPoP proofs for

> specific endpoints to use arbitrarily far into the future by choosing the

Reviewer: There's some ambiguity here which we can avoid by removing the "to use" part.
```suggestion
specific endpoints arbitrarily far into the future by choosing the
```

## Signed JWT Swapping

> Servers accepting signed DPoP proof JWTs MUST check the `typ` field is `dpop+jwt` in the

Reviewer: ```suggestion
Servers accepting signed DPoP proof JWTs MUST verify that the `typ` field is `dpop+jwt` in the
```
